### PR TITLE
Added support for new interfaces syntax

### PR DESCRIPTION
**Pull request by uweseimet**
@rdmark Can you please test?

```
  SCBR  Supported LUN numbers: 0-31
        Parameter support
        Default parameters: inet=10.10.20.1/24
                            interface=eth0
  SCDP  Supported LUN numbers: 0-31
        Parameter support
        Default parameters: inet=10.10.20.1/24
                            interface=eth0
```
```
>rascsi -ID6 -t scdp "interface=eth1:inet=1.1.1.1/24

  6:0  SCDP  Dayna:SCSI/Link:1.4a  inet=1.1.1.1/24:interface=eth1
```

### CODE REVIEW

**Comment by uweseimet** [comment]
@rdmark I just noticed that the netmask depends on the IP address. Looks as if it never was an indepedent parameter.
```
				if (mask == "8") {
					netmask = "255.0.0.0";
				}
				else if (mask == "16") {
					netmask = "255.255.0.0";
				}
				else if (mask == "24") {
					netmask = "255.255.255.0";
				}
				else {
					LOGERROR("Error: Invalid netmask in %s", bridge_ip.c_str());

					close(m_hTAP);
					close(ip_fd);
					close(br_socket_fd);
					return false;
				}
```
This means only the inet address should be configurable, just like before.

**Comment by rdmark** [comment]
@uweseimet I'm good with keeping inet and mask together in one parameter. Been testing the latest code for a bit and it seems to work as expected. Here's how the Web UI interprets the new default parameters for network devices.

Screenshot 2022-02-19 at 14-50-19 RaSCSI Control Page

**Comment by uweseimet** [comment]
@rdmark Looks good. Only having 2 parameters has the advantage of needing less space on the screen. For the web ui this is not that relevant, but for a smartphone it makes a difference.
But it may just be a matter of time until we will have more than 2 parameters (elsewhere) ;-).

File: doc/rascsi.1 [hunk 1]
```diff
@@ -84,7 +84,7 @@ Overrides the default locale for client-faces error messages. The client can ove
 .BR \-ID\fIn[:u] " " \fIFILE
 n is the SCSI ID number (0-7). u (0-31) is the optional LUN (logical unit). The default LUN is 0.
 .IP
-FILE is the name of the image file to use for the SCSI device. For devices that do not support an image file (SCBR, SCDP, SCLP, SCHS) the filename may have a special meaning or a dummy name can be provided. For SCBR and SCDP it is an optioinal prioritized list of network interfaces, e.g. "interfaces=eth0,eth1,wlan0". For SCLP it is the print command to be used and a reservation timeout in seconds, e.g. "cmd=lp -oraw:timeout=60".
+FILE is the name of the image file to use for the SCSI device. For devices that do not support an image file (SCBR, SCDP, SCLP, SCHS) the filename may have a special meaning or a dummy name can be provided. For SCBR and SCDP it is an optioinal prioritized list of network interfaces, an optional IP address and netmask, e.g. "interfaces=eth0,eth1,wlan0:inet=10.10.20.1/24". For SCLP it is the print command to be used and a reservation timeout in seconds, e.g. "cmd=lp -oraw:timeout=60".
```
Comment (rdmark) [inline doc/rascsi.1:87]: Shall we add '%f' to the printer cmd parameter example, since it is mandatory?

File: doc/rascsi_man_page.txt [hunk 2]
```diff
@@ -111,10 +111,10 @@ OPTIONS
...
               SCHS) the filename may have a special meaning or  a  dummy  name
               can  be  provided.  For SCBR and SCDP it is an optioinal priori‐
-              tized    list    of    network    interfaces,    e.g.    "inter‐
-              faces=eth0,eth1,wlan0".  For  SCLP it is the print command to be
-              used  and  a  reservation  timeout  in  seconds,  e.g.   "cmd=lp
-              -oraw:timeout=60".
+              tized list of network interfaces, an  optional  IP  address  and
+              netmask,  e.g.  "interfaces=eth0,eth1,wlan0:inet=10.10.20.1/24".
+              For SCLP it is the print command to be used  and  a  reservation
+              timeout in seconds, e.g. "cmd=lp -oraw:timeout=60".
```
Comment (rdmark) [inline doc/rascsi_man_page.txt:117]: Same as the above.

**Review by rdmark — APPROVED** [review]

File: doc/rascsi_man_page.txt (hunk 2 above)
Comment (uweseimet) [reply doc/rascsi_man_page.txt]: Done. rascsi_man_page.txt is generated from rascsi.1.

**Pull request merged**